app/javascript/miq-redux/middleware.js

```diff
@@ -5,10 +5,11 @@ import promiseMiddleware from 'redux-promise-middleware';
 export const taggingMiddleware = store => next => action => {
   const { type, meta, tagCategory, tag } = action;
   if (meta && meta.url) {
```

Comment:
```javascript
switch (type) {
  case 'UI-COMPONENTS_TAGGING_TOGGLE_TAG_VALUE_CHANGE':
    $.post(meta.url, meta.params(meta.type, store.getState(), tag));
    break;
  case 'UI-COMPONENTS_TAGGING_DELETE_ASSIGNED_TAG':
    $.post(meta.url, meta.onDelete(meta.type, params, tag.tagValue.id));
    break;
}
```

and move that `check: 0` to onDelete?

Reply:
Or ideally call the same method, since you're passing the type anyway.

The point is, there should be only 1 place dealing with this logic, not 2, or 3.